keyboards/keebio/iris/keymaps/xrthat/rules.mk

```diff
@@ -0,0 +1,6 @@
+RGBLIGHT_ENABLE = yes
```

Comment:
Most likely you can remove this entire file, in fact, as the base Iris rules.mk has RGBLight and Backlight enabled already.